repose-aggregator/components/filters/keystone-v2-filter/src/main/scala/org/openrepose/filters/keystonev2/KeystoneRequestHandler.scala

```diff
@@ -146,7 +146,7 @@ class KeystoneRequestHandler(identityServiceUri: String, akkaServiceClient: Akka
 
     val akkaResponse = Try(akkaServiceClient.get(
       s"$TOKEN_KEY_PREFIX$validatableToken",
-      s"$identityServiceUri$TOKEN_ENDPOINT/$validatableToken",
+      s"$identityServiceUri$TOKEN_ENDPOINT/$validatableToken${getApplyRcnRoles(applyRcnRoles)}",
```

Comment:
I don't really find either way to be better than the other. I do like that the first few characters of each parameter line up using interpolation, though. In the end, though, it wouldn't matter much to me.